include/backend/drm/properties.h

```diff
@@ -18,8 +18,9 @@ union wlr_drm_connector_props {
 		// atomic-modesetting only
 
 		uint32_t crtc_id;
+		uint32_t link_status;
```

Comment:
Thus is not an atomic-only property, so it should be placed above the above comment.

Reply:
Right. Should be fixed.